src/components/command-bar/commands/search/components/dialog-body/index.tsx

```diff
@@ -166,7 +136,20 @@ const SearchCommandBarDialogBodyContent = ({
 							icon={icon}
 							key={contentType}
 						>
-							{content}
+							{renderContent({
+								currentProductTag,
+								suggestedPages,
+								noResultsMessageSlot: (
```

Comment:
Previously the `<NoResultsMessage />` was rendered in the same way within each `<[ContentType]Contents />` component. We now need to pass some data into the no results message, to call out which tabs have results as the user searches.

Rather than thread that data through each content-type component separately, I switched to a slot interface. You'll see this `noResultsMessageSlot` change in each of the `<[ContentType]Contents />` components as well.

Reply:
I like it! The slots seem to be a great pattern.